### PR TITLE
Middleware Configuration

### PR DESCRIPTION
**Pull request by jawrainey**
We've [setup the middleware](https://github.com/ideafast/middleware-services/commit/6eb4a7a3e2dcde5f0c28e2e74d068cf7cde18013) so that we can build/tag/deploy an image based on a specific checksum (via [git tag](https://github.com/ideafast/middleware-services/tags) and tagged images on [Docker Hub](https://hub.docker.com/r/ideafast/middleware/tags)). 

The initial version of the middleware is now been deployed using the compose-file below. 

**To Test** 

- Verify no changes exist on the server in /middleware/
- Validate that the live/dev APIs are live/working at: [live API](https://api.wp3.ideafast.eu/devices), [dev API](https://dev.api.wp3.ideafast.eu/devices).

### CODE REVIEW

File: instances/middleware/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  consumer_dev:
+    image: ideafast/middleware:0.1.0
+    networks:
+      - web
+    container_name: consumer_dev
+    env_file:
+      - .consumer.dev.env
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.consumer_dev.rule=Host(`dev.api.wp3.ideafast.eu`)"
```
Comment (farzanehfar) [inline instances/middleware/docker-compose.yml:30]: Good Job on hosting dev.api.wp3.ideafast.eu!

File: instances/middleware/.consumer.env.example [hunk 2]
```diff
@@ -0,0 +1,7 @@
+INVENTORY_BASE_URL=
+INVENTORY_TOKEN=
+MODULE_NAME=consumer.main
+VARIABLE_NAME=consumer
+LOG_LEVEL=INFO
+SUPPORT_BASE_URL=
+SUPPORT_TOKEN=
```
Comment (farzanehfar) [inline instances/middleware/.consumer.env.example:7]: Great to see Specific Token per Support and Inventory.

**Review by farzanehfar — APPROVED** [review]

**Review by drewmiley — COMMENTED** [review]
Do we need .dtransfer.env.example?

**Comment by jawrainey** [comment]
@drewmiley -- great point! Since we have not yet deployed the `data_transfer` container, we could do that in another PR once we add that image, etc? I suspect that .dtransfer.env.example will change once we have DMP and devices setup -- likely it will be a very long example file 👍🏼

**Pull request merged**